src/mlpack/tests/akfn_test.cpp

```diff
@@ -166,7 +166,8 @@ TEST_CASE("AKFNSingleCoverTreeTest", "[AKFNTest]")
 TEST_CASE("AKFNDualCoverTreeTest", "[AKFNTest]")
 {
   arma::mat dataset;
-  data::Load("test_data_3_1000.csv", dataset);
+  if (!data::Load("test_data_3_1000.csv", dataset))
+    FAIL("Unable to load data test_data_3_1000.csv!");
```

Comment:
Do you think instead of `Unable to load data` we could use `Unable to load dataset` for one this is what we use in other cases already and since the dataset is called `test_data_3_1000` it might avoid some confusion?